### PR TITLE
fix for markdown image plugin

### PR DESCRIPTION
**Pull request by logan-anderson**
@dwalkr Here is my fix if you want to see it for reference

### CODE REVIEW

**Review by dwalkr — APPROVED** [review]

**Comment by logan-anderson** [comment]
@dwalkr I see two issues.
1. For some reason media in the top level is not listed
2. Drag and drop does not work


see https://www.loom.com/share/cd9b234570f740ad86e9d637a911e7d7

**Comment by dwalkr** [comment]
> @dwalkr I see two issues.
> 
> 1. For some reason media in the top level is not listed
> 2. Drag and drop does not work
> 
> see https://www.loom.com/share/cd9b234570f740ad86e9d637a911e7d7

#/1 is fixed in 760366cd8c1f36439f44f4e358e696d53871f4a8
Let's wait until Jack's drag+drop PR is merged before we determine if #/2 is still a problem / how to fix

**Review by logan-anderson — COMMENTED** [review]
Looks good to me! I cant approve it as I made the PR

**Review by jeffsee55 — APPROVED** [review]

**Pull request merged**
